pkg/openthread/contrib/platform_uart.c

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) 2016 José Ignacio Alamos
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ * @author  José Ignacio Alamos <jialamos@uc.cl>
+ */
+
+#include <stdint.h>
+#include <stdio.h>
+
+#include <openthread/platform/uart.h>
+
+/* OpenThread will call this for enabling UART (required for OpenThread's CLI)*/
+ThreadError otPlatUartEnable(void)
+{
+    return kThreadError_None;
+}
+
+/* OpenThread will call this for disabling UART */
+ThreadError otPlatUartDisable(void)
+{
+    return kThreadError_None;
+}
+
+/* OpenThread will call this for sending data through UART */
+ThreadError otPlatUartSend(const uint8_t *aBuf, uint16_t aBufLength)
```

Comment:
Sort of. Whatever OT has to print, it will be sent through UART. For instance, all shell commands are sent and received with otPlatUartSend and otPlatUartReceived. For setting up the Border Router, it's possible to do something similar to ethos or using another UART port.
Since I only needed the CLI output, I decided to print it :)

Cheers